drivers/storage/efs/storage/efs_storage.go

```diff
@@ -604,51 +796,43 @@ func (d *driver) getVolumeAttachments(
 	return atts, nil
 }
 
-func (d *driver) efsClient() *awsefs.EFS {
-	config := aws.NewConfig().
-		WithCredentials(d.awsCreds).
-		WithRegion(d.region())
-
-	if types.Debug {
-		config = config.
-			WithLogger(newAwsLogger()).
-			WithLogLevel(aws.LogDebug)
-	}
+// Retrieve config arguments
+func (d *driver) getAccessKey() string {
+	return d.config.GetString(efs.ConfigEFSAccessKey)
+}
 
-	return awsefs.New(session.New(), config)
+func (d *driver) secretKey() string {
```

Comment:
This should be probably `getSecretKey()` to follow the convention of other method names.